idp.go

```diff
@@ -157,6 +185,14 @@ func (idp *IdentityProvider) AddAttribute(name string, value string, format stri
 	})
 }
 
+func (idp *IdentityProvider) AddContactPerson(service MetadataBinding) {
```

Comment:
@sccalabr AddContactPerson function should add ContactPersion, not MetadataBinding